src/WebUI/src/middlewares/auth.ts

```diff
@@ -47,12 +49,28 @@ export const authRouterMiddleware: NavigationGuard = async to => {
   return true;
 };
 
+const errorHandler = (error: unknown) => {
+  if (error instanceof ErrorResponse && error?.error === 'invalid_grant') {
+    return { name: 'Banned' };
+  }
```

Comment:
invalid_grant can mean something else. I've changed the error to access_denied (https://github.com/verdie-g/crpg/commit/82e2f7eab17decd61d82a0a735b83add92b48ba9)